README.md

```diff
@@ -8,10 +8,24 @@ sudo apt-get install freetds-dev
 pew new mf_dataentry
 # for more info about pymssql check ffapb/ffa-blotter requirement.txt
 pip install django progressbar33 git+https://github.com/pymssql/pymssql.git
+pip install pip PyYAML
++pip install Flask-SQLAlchemy
```

Comment:
what is this for?

Reply:
it's for orm library. when you started using orm library, I am obliged to install FLASk-SQLAlchemy in order to the project

Reply:
I think you just need `SQLAlchemy` instead